app/src/main/java/eu/kanade/tachiyomi/data/backup/BackupRestoreService.kt

```diff
@@ -65,7 +66,12 @@ class BackupRestoreService : Service() {
          * @return true if the service is running, false otherwise.
          */
         fun isRunning(context: Context): Boolean =
+            context.isServiceRunning(BackupRestoreService::class.java) /* SY --> */ || context.isServiceRunning(FullBackupRestoreService::class.java) /* SY <-- */
```

Comment:
I'm not sure if we should actually have 2 services rather than 2 logic implementations used within the service, based on flags similar to what you have now, i.e. just shifting the decision on which service logic to use to within the service itself.